src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -1115,6 +1115,7 @@ pbs_python_setup_queue_class_attributes(void)
 			if (!py_default_value) {
 				/* TODO, continuing instead of fatal error */
 				log_err(-1, attr_def_p->at_name, "could not set default value");
+				attr_def_p++;
```

Comment:
If we enter this failure check, we want to continue instead of throwing an error. So we want to proceed to set the default values for the next parameter and so on. We need to increment this pointer to set the value for the next parameter. I hope I was able to explain it right.

Reply:
there is another continue above this, please check, if we need to do the same there as well...